docs/create_products.rst

```diff
@@ -0,0 +1,74 @@
+.. _create_products:
+
+Create Jdaviz-readable Products
+===============================
+
+Data products can be loaded in the different Jdaviz configurations using
+various methods, e.g. loading :class:`~specutils.Spectrum1D` objects, loading
+from fits files, or loading a set of fits files from a directory.
+Here, we list a few ways in which data can be packaged to be easily loaded
+into a ``jdaviz`` configuration.
+
+Data in a database
+------------------
+
+If the data are stored in a database, we recommend storing a :class:`~specutils.Spectrum1D` object
+per entry. This would allow the user to query the data and visualize it in
+Jdaviz with few lines of code. This is an example with an imaginary database.
+
+.. code-block:: python
+
+    from database import Client
+    from jdaviz import Specviz
+
+    client = Client()
+    search = client.find(**options**)
+    results = client.retrieve(search)
+    spec1d = results[0]['Spectrum1D']
+
+    specviz = Specviz()
+    specviz.load_data(spec1d, data_label='my spectrum')
+    specviz.show()
+
+
+Data in fits files
```

Comment:
FITS is an acronym. Same comment applies to all similar occurrences.
```suggestion
Data in FITS files
```

Reply:
thanks, i will correct the rest.